tests/error/inout_errors/override_after_call.err

```diff
@@ -0,0 +1,7 @@
+Guppy compilation failed. Error in file $FILE:16
+
+14:    @guppy(module)
+15:    def test(q1: qubit, q2: qubit) -> tuple[qubit, qubit]:
+16:        q1 = foo(q1, q2)
+                    ^^
+GuppyError: Variable `q1` with linear type `qubit` is not used
```

Comment:
This error is quite confusing. We need a special case for unused variables if the last assignment came implicitly from an inout. I'll try to adress that in a separate PR later (see #318).

Any ideas what the message should be here?